Apps/WebClient/src/ClientApp/src/components/services/OrganDonorDetailsCard.vue

```diff
@@ -79,10 +115,13 @@ function getDecisionFile(): void {
 function showConfirmationModal(): void {
     sensitiveDocumentModal.value?.showModal();
 }
+
+canAccessOrganDonorRegistration(true);
```

Comment:
Is there any need to have the hardcoded `showDatasetWarning`? 

If the flag is necessary why not have `showDatasetWarning` default to true and remove the hardcoded value?

Reply:
Yes it is required.  Changed default to true.